certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -2242,6 +2242,74 @@
         },
         "deprecated": false,
         "x-ms-visibility": "important"
+      },
+      "get": {
+        "tags": [
+          "DocuSign"
+        ],
+        "summary": "Get info for recipient tab",
+        "description": "Returns the value of the tab for the given recipient",
+        "operationId": "GetTabValues",
```

Comment:
I would name it as `GetTabInfo` , that way in future if we want to return fields other than value, the name still makes sense.